### PR TITLE
JavaDriver

### PR DESCRIPTION
**Pull request by darroyocazorla**

### CODE REVIEW

**Comment by jjlopezm** [comment]
+1

File: crossdata-driver/src/main/scala/com/stratio/crossdata/driver/JavaDriver.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+package com.stratio.crossdata.driver
+
+import akka.util.Timeout
+import com.stratio.crossdata.common.metadata.FieldMetadata
+import com.stratio.crossdata.common.{SQLCommand, SQLResult}
+import org.apache.log4j.Logger
+
+import scala.collection.JavaConversions._
+
+class JavaDriver(seedNodes: Option[java.util.List[String]] = None) {
```
Comment (miguel0afd) [inline crossdata-driver/src/main/scala/com/stratio/crossdata/driver/JavaDriver.scala]: Class constructor for Java shouldn't use Option and Default value

File: crossdata-driver/src/main/scala/com/stratio/crossdata/driver/JavaDriver.scala [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  def listTables(database: String): java.util.List[String] = {
+    scalaDriver.listTables(Some(database))
+  }
+
+  def describeTable(database: String, tableName: String): java.util.List[FieldMetadata] = {
+    scalaDriver.describeTable(Some(database), tableName)
+  }
+
+  def describeTable( tableName: String): java.util.List[FieldMetadata] = {
```
Comment (miguel0afd) [inline crossdata-driver/src/main/scala/com/stratio/crossdata/driver/JavaDriver.scala]: Please, remove the space before the tableName

**Comment by miguel0afd** [comment]
+1 LGTM

**Pull request merged**